### ISSUE TITLE
Manual Registration in android devices

### ISSUE THREAD

**Issue by nitin-787** [nitin-787/mynotes#97]
### Screenshots

https://user-images.githubusercontent.com/76255199/194717769-761f058d-8d91-4e4e-a29c-274d4d2b79eb.mp4

**Comment by SwaroopSambhayya** [comment]
Hi, I would like to take up this!

**Comment by nitin-787** [comment]
> Hi, I would like to take up this!

go ahead

**Comment by SwaroopSambhayya** [comment]
can you assign?

**Comment by vishalkevin11** [comment]
Hi @SwaroopSambhayya ,

Had been debugging this issue and I clearly see `context.read<AuthBloc>().add(const AuthEventInitialize());` in build method is the culprit. 
My Analysis
- Build method is called multiple time
- AuthBloc has `on<AuthEventInitialize>((event, emit) ` registered
- Since User details in null during login/signup, The event `AuthStateLoggedOut` is a called everytime causing `LoginPage` to be shown.
```
 if (user == null) {
        emit(
          const AuthStateLoggedOut(
            exception: null,
            isLoading: false,
          ),
        );
   }

```
My Approach
- Since User details are null during login/signup, I would maintain one more state for knowing is it is login screen / signup screen. So I don't end up triggering `AuthStateLoggedOut(exception: null, isLoading: false,)` when I am on login/ signup screen.

<img width="575" alt="Screenshot 2022-10-13 at 9 29 12 AM" src="https://user-images.githubusercontent.com/9132837/195496032-01270276-554a-4666-98af-fdee2a64496d.png">

Hope this will helps.

**Comment by SwaroopSambhayya** [comment]
Hi Nitin, i have created PR #104 please review

**Comment by nitin-787** [comment]
### fix #104

**Comment by nitin-787** [comment]
thanks for contribution @vishalkevin11 and @SwaroopSambhayya
